content/en/security/threats/security_signals.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Investigate Security Signals
+kind: documentation
+disable_toc: false
+further_reading:
+  - link: "/security/default_rules/?category=cat-csm-threats#all"
+    tag: "Documentation"
+    text: "Explore CSM Threats detection rules"
+  - link: "/security/threats/workload_security_rules"
+    tag: "Documentation"
+    text: "Learn how to manage CSM Threats detection rules"
+  - link: "/security/notifications/"
+    tag: "Documentation"
+    text: "Learn more about security notifications"
+---
+
+[Cloud Security Management Threats][9] (CSM Threats) security signals are created when Datadog detects a threat based on a security rule. View, search, filter, and investigate security signals in the [Threats Explorer][4], or configure [Notification Rules][1] to send signals to third-party tools.
+
+To modify security signals, you must have the `security_monitoring_signals_write` permission. See [Role Based Access Control][3] for more information about Datadog's default roles and granular role-based access control permissions available for Cloud Security Management.
+
+{{< img src="security/cws/csm_threats_signals.png" alt="CSM Threats security signals page" width="100%">}}
+
+## Filter security signals
+
+To filter the security signals in the [Threats Explorer][4]:
+
+1. Click the **Options** button and add the column **Signal State**.
+2. Add the facet `@workflow.triage.state`. This displays signal status and enables you to sort by status in the table header.
```

Comment:
```suggestion
2. Add the facet `@workflow.triage.state`. This displays signal status and enables you to sort by status in the table header.
```
This isn't entirely correct, this allows you to filter the Signal State by "open" or "under review". 
https://a.cl.ly/bLuydeom

Reply:
Archived appears but only if you've archived signals within the selected timeframe.